docs/enterprise/active-learning/active_learning.md

```diff
@@ -54,9 +83,50 @@ preservation) of images before submission into Roboflow platform (optional)
 * `batching_strategy`: configuration of labeling batches creation - details below (required if `enabled`)
 * `tags`: list of tags (each contains 1-64 characters from range `a-z, A-Z, 0-9, and -_:/.[]<>{}@`) (optional)
 
-## How to Configure Active Learning
+### Batching strategy
+
+The `batching_strategy` field holds a dictionary with the following configuration options:
+
+* `batches_name_prefix`: A string representing the prefix of batches names created by Active Learning (required)
+* `recreation_interval`: One of value `["never", "daily", "weekly", "monthly"]`: representing the interval which is to be used to create separate batches - thanks to that - user can control the flow of labeling batches in time (required).
+* `max_batch_images`: Positive integer representing maximum size of batch (applied on top of any strategy limits) to prevent too much data to be collected (optional)
```

Comment:
```suggestion
* `max_batch_images`: Positive integer representing the maximum size of the batch (applied on top of any strategy limits) to prevent too much data from being collected (optional)
```